corehq/apps/users/forms.py

```diff
@@ -1270,6 +1276,12 @@ def clean_search_string(self):
 
     def clean_domains(self):
         domains = self.data.getlist('domains[]', [self.domain])
+        try:
+            is_all_domain_download = self.data['is_all_domain_download']
+            if is_all_domain_download == 'true' or is_all_domain_download == 'on':
+                domains = DomainPermissionsMirror.mirror_domains(self.domain)
+        except MultiValueDictKeyError:
```

Comment:
I think it's a little more typical to check `'is_all_domain_download' in self.data` rather than a try/except, because it isn't really an exceptional condition, it's one of two normal code paths.